packages/fxa-auth-server/lib/senders/email.js

```diff
@@ -1268,7 +1268,7 @@ module.exports = function (log, config, oauthdb) {
       email: message.email,
       uid: message.uid,
     });
-
+    const onDesktopOrTabletDevice = !message.onMobileDevice;
```

Comment:
I needed to reverse this here or else the `appBadges.html` `if`/`else` got messy to keep `cadReminders` rendering “connect another device” instead of “connect another desktop device.”

I didn’t pass this in as `onDesktopOrTabletDevice` because reversing the boolean in `postVerifyEmail` instead makes testing for the two different versions of copy much easier.